doc/api/cli.md

```diff
@@ -935,7 +935,7 @@ in the `$schema` must be replaced with the version of Node.js you are using.
 
 ```json
 {
-  "$schema": "https://nodejs.org/dist/vX.Y.Z/docs/node-config-schema.json",
+  "$schema": "https://nodejs.org/docs/vX.Y.Z/node-config-schema.json",
```

Comment:
https://nodejs.org/dist/v23.10.0/docs/node-config-schema.json does exist, though.

Reply:
Huh, you’re right. Both work. Then I’m not sure what I tried before that didn’t work.

Feel free to approve or close this PR as you see fit.